cmd/root.go

```diff
@@ -167,6 +174,11 @@ func runRoot(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if chartVersion != "" && chartName == "" {
+		logger.Printf("error: chart Version depends on a chart name, please specify one")
```

Comment:
So in this line I'm only ensuring, that if the user sets a version, we need the name of the chart.

Overall we have this scenarios:
- Get latest charts (no filters)
- Get all charts (`--all-versions`)
- Get latest named chart (`--chart-name`)
- Get all versions of named chart (`--chart-name && --all-versions`)
- Get specific version of named chart (`--chart-name && --chart-version`)

Basically if you set `--all-versions` flag, it WILL ignore the `--chart-version` flag but NOT the `chart-name` flag.

Reply:
You can see that here
https://github.com/openSUSE/helm-mirror/pull/20/files#diff-0bf75882225f9337e97650545c58545fR72

Reply:
Oh yes, sorry... my fault